data/js/show_data.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+    var button = document.querySelector('button');
+
+    addon.port.on('history', function(history) {
+        for(var x = 0, y = history.length; x < y; x++) {
+			document.querySelector('#history' + (x+1)).innerHTML = history[x];
+		}
+    });
+
+    addon.port.on('bookmarks', function(bookmarks) {
+        for(var x = 0, y = bookmarks.length; x < y; x++) {
+            document.querySelector('#bookmarks' + (x+1)).innerHTML = bookmarks[x];
+		}
```

Comment:
Ooooh, definitely use spaces rather than tabs. Which editor are you using? There is an .editorconfig in the repo that should set all those things for you is using Atom or Sublime